CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute to CTF Handbooks
+
+## Making Changes to the Handbooks
+
+### Structure of the Writerside Folder
+```
+./Writerside/
+├───cfg/
+│   ├───buildprofiles.xml
+├───images/
+│   ├───air.png
+│   ├───appenditemname.png
+│   ├───...
+├───topics/
+│   ├───Commonly-Used-Itemstrings-List.md
+│   ├───Making-a-Map-for-Capture-the-Flag.md
+│   └───...
+├───c.list
+├───ch.tree
+├───v.list
+└───writerside.cfg
+```
+The main folders you would need to know are the [`images`](./Writerside/images) and [`topics`](./Writerside/topics) sub-folders within the [`Writerside`](./Writerside) folder.
+- [`images`](./Writerside/images): This folder contains all the image media used in the handbook. When adding new images, place them in this folder.
+- [`topics`](./Writerside/topics): This folder is the home to all the Handbook markdown files. When adding new chapters [without using Writerside](#without-using-writerside), make new markdown files for them here.
+
+---
+
+The below sections will introduce you to the two ways in which you can make changes to the content of handbooks: (1) [Using Writerside](#using-writerside) and (2) [Without using Writerside](#without-using-writerside).
+
+### Using Writerside
+1. Install [JetBrains Writerside](https://www.jetbrains.com/writerside/) (version `2023.3`, build `233.14938`) *OR* [Writerside IDE Plugin](https://plugins.jetbrains.com/plugin/20158-writerside?_gl=1%2All6kto%2A_ga%2ANDc2OTczNjcxLjE3MTIyNDEwNzM.%2A_ga_9J976DJZ68%2AMTcxMjI0ODM2Ny4yLjEuMTcxMjI0ODgwOS40Ny4wLjA.) (version `2024.03.233.14938`) on a compatible JetBrains IDE.
```

Comment:
Why not just the latest version? I wonder if Writerside has an LTS; probably not.

Reply:
I did that to prevent any versions that we might've not updated on our machines yet. Another alternative is to mention the latest version and then make them mention it in the PR so we can double-check.

Reply:
Which option do we choose then?

Reply:
I'd prefer mentioning versions, or else I don't feel writing about mentioning the version in the PR in CONTRIBUTING.md is suitable. We might have to make a PR template. Difficult to decide lol

Reply:
Alright then 👍

Reply:
Ok, marking this as resolved.